fiftyone/server/utils/json/jsonpatch/methods.py

```
@@ -144,12 +144,13 @@ def add(
-            except AttributeError as err:
-                raise err
+            # try:
+            setattr(target, name, value)
+            # except AttributeError as err:
+            #     raise err
 
     except Exception as err:
+        print(err)
         raise ValueError(
             f"Unable to add value with path: {pointer.path}"
         ) from err
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)